src/Components/Courses/Courses.jsx

```diff
@@ -0,0 +1,77 @@
+import React from "react";
+import { Link } from "react-router-dom";
+import "./courses.scss";
+import book from "../../Assets/Images/book.png";
+import bookmark from "../../Assets/Images/bookmark.png";
+import backArrow from "../../Assets/Images/arrow.png";
+
+const Courses = () => {
+  const coursesData = [
+    {
+      insti: "National Institute of Technology, Silchar",
+      subjectName: "Chemistry",
+    },
+    {
+      insti: "National Institute of Technology, Warangal",
+      subjectName: "Chemistry",
+    },
+    {
+      insti: "Indian Institute of Technology, Delhi",
+      subjectName: "Physics",
+    },
+    {
+      insti: "Indian Institute of Technology, Bombay",
+      subjectName: "Physics",
+    },
+    {
+      insti: "Indian Institute of Technology, Bombay",
+      subjectName: "Chemistry",
+    },
+    {
+      insti: "Indian Institute of Technology, Bombay",
+      subjectName: "Maths",
+    },
+    {
+      insti: "Indian Institute of Technology, Bombay",
+      subjectName: "Physics",
+    },
+    {
+      insti: "Indian Institute of Technology, Bombay",
+      subjectName: "Chemistry",
+    },
+    {
+      insti: "Indian Institute of Technology, Bombay",
+      subjectName: "Maths",
+    },
+    {
+      insti: "Indian Institute of Technology, Bombay",
+      subjectName: "Physics",
+    },
+  ];
```

Comment:
Use json file in public and fetch API to mimic backend API call for your ease later on after backend integration

Reply:
Made this change in the latest commit.